tests/query_tests.rs

```diff
@@ -1,38 +0,0 @@
-use assert_cmd::prelude::*; // Add methods on commands
```

Comment:
Why delete this file?

Reply:
I had thought it was meant to be on your branch and I didn't want to push code that was yours.  But now it is on master and I've rebased with master and so the query_tests file is back.